packages/launchpad-ledger/src/ledgersigner.ts

```diff
@@ -1,18 +1,33 @@
-import { AccountData, encodeSecp256k1Signature, OfflineSigner, StdSignature } from "@cosmjs/launchpad";
-
-import { LaunchpadLedger, LaunchpadLedgerOptions } from "./launchpadledger";
+import {
+  AccountData,
+  encodeSecp256k1Signature,
+  makeCosmoshubPath,
+  OfflineSigner,
+  StdSignature,
+} from "@cosmjs/launchpad";
+
+import { LaunchpadLedger } from "./launchpadledger";
+
+export interface LedgerSignerOptions {
+  readonly accountNumbers?: readonly number[];
```

Comment:
Instead of account numbers we should allow `HdPath[]` here (falling back to `[makeComsoshubPath(0)]`). Then the demo code can set a bunch of paths.